custom_components/nissan_leaf_obd_ble/elm327.py

```diff
@@ -106,7 +106,10 @@ async def create(
         start_low_power=False,
     ):
         """Initialize ELM327."""
-        uuids = [u.lower() for u in (device.metadata or {}).get("uuids", [])]
+        try:
+            uuids = [u.lower() for u in device.details['props']['UUIDs']]
+        except KeyError:
+            uuids = [""]
```

Comment:
I wasn't sure what the best default was for this, or if you think it'd make more sense to not catch the exception here (since it'd mean something was broken anyway).